Specs/Number/Italian/NumberRangeModel.json

```diff
@@ -6,464 +6,464 @@
 }
 ,
 {
-  "Input": "This number is larger than twenty and less or equal than thirty five.",
+  "Input": "Questo numero è più grande di venti e minore o uguale a trentacinque.",
   "NotSupported": "javascript",
   "NotSupportedByDesign": "python",
   "Results": [
     {
-      "Text": "larger than twenty and less or equal than thirty five",
+      "Text": "più grande di venti e minore o uguale a trentacinque",
       "TypeName": "numberrange",
       "Resolution": {
-        "value": "(20,35]"
```

Comment:
The interval syntax should not be modified by language.
This is a math syntax, it can represent intervals like [-15,-1], for example.

Reply:
It would be ambiguous for the case of open intervals with negative bounds.